package.json

```diff
@@ -189,5 +190,12 @@
       "name": "alesun",
       "email": "alesun@exadel.com"
     }
+  ],
+  "eslint": [
+    {
+      "name": "eslint-plugin-custom-rules",
+      "version": "1.0.0",
+      "main": "index.js"
```

Comment:
It looks like meta information for lint module is not correct.
"main" should targeting init file for all created lint rules, which is "eslint/index.js"
Regarding "version" I think it have sense to make it equal to actual library version.